src/ServiceControl.Persistence.RavenDB/RavenPersistence.cs

```diff
@@ -27,6 +27,8 @@ public void AddPersistence(IServiceCollection services)
                 return;
             }
 
+            services.AddSingleton<INewFeatureDataStore, RavenNewFeatureDataStore>();
```

Comment:
We register the implementation in the Raven persistence.
This is where I think we may have issues because this registration is in a new AssemblyLoadContext